R/drive_upload.R

```diff
@@ -179,17 +176,17 @@ build_drive_upload <- function(input = NULL,
     id <- proc_res$id
   }
 
-  url <- file.path(.drive$base_url,
-                   "upload/drive/v3/files",
-                   paste0(id, "?uploadType=media"))
-
-  list(
-    method = "PATCH",
-    url = url,
+  build_request(
+    path = "/upload/drive/v3/files/{fileId}",
```

Comment:
there are 2 endpoints that have 2 different paths depending on what you are trying to do (both have to do with uploads). If you are just updating the metadata, you use the path "/drive/v3/files/{fileId}" (that is what we are doing most of the time), but if you are updating the *actual* file (like upload new content, as we are here), you have to change the path to "/*upload*/drive/v3/files/{fileId}"